tests/requirement/test_requirement_endo_bleeding.py

```
@@ -0,0 +1,64 @@
+    def test_high_bleed_risk_endo_intervention(self):
+        """
+        Test if the requirement for high bleeding risk endoscopy intervention is created correctly.
+        """
+        requirement = Requirement.objects.get(name=self.req_name_bleeding_high)
+        self.assertIsInstance(requirement, Requirement)
+        logger.info(f"Requirement for high bleeding risk endoscopy intervention: {requirement.name}") 
+        self.assertEqual(requirement.name, self.req_name_bleeding_high)
+
+        finding_interventions = requirement.finding_interventions.all()
+
+        intervention_names = [_.name for _ in finding_interventions]
```

Ruff (0.8.2)

Local variable `intervention_names` is assigned to but never used

Remove assignment to unused variable `intervention_names`

(F841)